.github/workflows/docs.yml

```diff
@@ -29,6 +30,32 @@ jobs:
           cp ./python/*.md docs/python
           cp ./python/examples/README.md docs/python/examples.md
 
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          fail: false
+          # Only check docs
+          args: |
+            --base
+            .
+            --verbose
+            --no-progress
+            './*.md'
+            './docs/**/*.md'
+            './docs/**/*.html'
+            --exclude-path
```

Comment:
Is there an option to exclude external links?

Reply:
I think we'd want to check external links, but yes lots of options.

I'm trying to exclude the changelog, but not got that working yet

Reply:
We can, but they often fail because of anti-bot detections.